### PR TITLE
Multiple Choice view Enhancement

### PR DESCRIPTION
**Pull request by mehtadarshan1**
Question Edit view:
screen shot 2017-10-28 at 12 17 04 am

Question Creation view: 
screen shot 2017-10-28 at 12 17 26 am

Question View for Student:
screen shot 2017-10-28 at 12 17 51 am

### CODE REVIEW

**Comment by esteeroramy** [comment]
Why is the text 'right' and 'wrong' as opposed to 'true' and 'false'

**Comment by esteeroramy** [comment]
Will review this PR more thoroughly tomorrow

**Comment by mehtadarshan1** [comment]
That’s just the answer options... not related to code

File: public/js/mc-answer.js [hunk 1]
```diff
@@ -5,8 +5,6 @@ var mcAnswerCount = 4; // default number of options
...
-    newdiv.innerHTML = inputdiv.format(mcAnswerCount,mcAnswerCount,mcAnswerCount,mcAnswerCount);
-    $('#qAnswer > div.form-group').append(newdiv);
-}
+    var inputdiv = "<div class='row'><div class='col s1'><input type='radio' name='radbutton' value='mcans{0}' id='mcans{1}' required='required'><label for='mcans{2}'></label></div><div class='input-field col s10'><input class='form-control' type='text' name='mcans{3}' placeholder='Enter Answer Here' required='required' style='float:left;'></div><div class='input-field col s1'><a class='btn-floating btn-tiny waves-effect waves-light red' onclick='$(this).parent().parent().remove()'><i class='tiny material-icons'>close</i></a></div></div>";
+    $('#qAnswer > div.form-group.col.s12').append(inputdiv.format(mcAnswerCount,mcAnswerCount,mcAnswerCount,mcAnswerCount));
```
Comment (ahmedqarmout2) [inline public/js/mc-answer.js]: I changed the format function to take an array. so now all you have to do is use {0} for all of them, then pass [mcAnswerCount] to the format function

**Comment by ahmedqarmout2** [comment]
nice work. Please use the new format function, and if you could left align the choice in the question view that would be great.

**Review by ahmedqarmout2 — APPROVED** [review]
nice work fam

**Pull request merged**